pkg/dbcrypt/dbcrypt_test.go

```diff
@@ -50,7 +50,7 @@ func (a *MyTable) BeforeSave(tx *gorm.DB) (err error) {
 func (a *MyTable) AfterFind(tx *gorm.DB) (err error) {
 	err = cryptor.DecryptStruct(a)
 	if err != nil {
-		err := tx.AddError(fmt.Errorf("Unable to decrypt password %v", err))
+		err := tx.AddError(fmt.Errorf("unable to decrypt password %v", err))
```

Comment:
Since we're already here... :D
```suggestion
		err := tx.AddError(fmt.Errorf("unable to decrypt password %w", err))
```